src/main/java/http/routes/structured_data/HTMLResponse.java

```diff
@@ -17,8 +17,8 @@ public class HTMLResponse implements Route {
     final String textBody;
 
     public HTMLResponse() {
-        headers = Arrays.asList(ContentTypes.CONTENT_TYPE_HTML.getType(), Methods.GET.getMethod(), Methods.HEAD.getMethod());
-        headersResponse = String.format("%s%sAllow: %s, %s", headers.get(0), CRLF, headers.get(1), headers.get(2));
+        headers = Arrays.asList(Methods.GET.getMethod(), Methods.HEAD.getMethod());
```

Comment:
These two lines appear in all the `Route` classes. It would be good to remove duplication by creating a class that `ResponseHeader` which contains methods that return the strings.

Reply:
extracted to a `HTTPMethodsHeader` class